docs/specs/validation.yml

```diff
@@ -10,9 +10,12 @@ inputs:
     bar
     >>>>>>> cb1abc6bd98cfc84317f8aa95a7662815417802d
 outputs:
-  docs/a.json:
+  docs/a.json: |
+    {
+      "conceptual": ""
```

Comment:
Should we still have output content for this test?

Reply:
yes, we still have output, but we don't need to verify that